models/src/main/thrift/content/v1.thrift

```diff
@@ -901,6 +903,28 @@ struct ListElementFields {
     2: optional ListType type;
 }
 
+struct TimelineEvent {
+    1: required list<BlockElement> body = [];
+
+    2: optional BlockElement main;
+
+    3: optional string title;
+
+    4: optional string date;
+
+    5: optional string label;
```

Comment:
How is this label used? We already have a title, but it looks like there's additional description required, I wonder whether we can document why here

Reply:
In this example "Primary" is a label. 
<img width="295" alt="Screenshot 2024-03-28 at 15 54 21" src="https://github.com/guardian/content-api-models/assets/2619836/b628f1fd-8f99-4e9d-8342-74c49d0930ce">